Services/include/housekeeping/housekeeping_service.h

```diff
@@ -67,6 +67,7 @@ typedef struct __attribute__((packed)) {
     ADCS_HouseKeeping adcs_hk;            // ADCS housekeeping struct
     athena_housekeeping Athena_hk;        // Athena housekeeping struct
     eps_instantaneous_telemetry_t EPS_hk; // EPS telemetry struct
+    eps_startup_telemetry_t EPS_startup_hk;
```

Comment:
I believe this has been added to the beacon branch that is waiting to be merged!

Reply:
OK. Your changes should go in. I will remove these commits.